packages/compare-slider-block/src/components/SliderLine/SliderLine.tsx

```diff
@@ -0,0 +1,128 @@
+            : {
+                  display: !sliderWidthValue ? 'none' : 'block',
+                  height: sliderWidthValue + 2,
+                  background: isDark(sliderColor) ? 'white' : 'black',
+              };
+    };
+
+    const getHandleComponent = () => {
+        switch (handle) {
+            case Handle.Circles: {
+                return (
+                    <div
+                        className="tw-flex"
+                        style={{
+                            color: toRgbaString(sliderColor),
+                            gap: sliderWidth,
+                        }}
+                    >
+                        <div
+                            className={joinClassNames([
+                                `${isDark(sliderColor) ? 'tw-bg-white/[.8]' : 'tw-bg-black/[.7]'}`,
+                                'tw-h-[22px] tw-w-[22px] tw-flex tw-justify-center tw-items-center tw-pr-[2px] tw-rounded-full tw-mr-1',
+                            ])}
+                        >
+                            <IconCaretLeft16 />
+                        </div>
+                        <div
+                            className={joinClassNames([
+                                `${isDark(sliderColor) ? 'tw-bg-white/[.8]' : 'tw-bg-black/[.7]'}`,
+                                'tw-h-[22px] tw-w-[22px] tw-flex tw-justify-center tw-items-center tw-pl-[2px] tw-rounded-full tw-ml-1',
+                            ])}
+                        >
+                            <IconCaretRight16 />
+                        </div>
+                    </div>
+                );
+            }
+            case Handle.Arrows: {
+                return (
+                    <div
+                        className={joinClassNames([
+                            isDark(sliderColor) ? '[&_path]:tw-stroke-white' : '[&_path]:tw-stroke-black/[.7]',
+                            'tw-flex [&_path]:tw-stroke-[3px]',
+                        ])}
+                        style={{
+                            color: toRgbaString(sliderColor),
+                            gap: sliderWidth,
+                            paintOrder: 'stroke',
+                        }}
+                    >
+                        <IconCaretLeft32 />
+                        <IconCaretRight32 />
+                    </div>
+                );
+            }
+            default: {
+                return <div />;
+            }
+        }
+    };
+
+    return (
+        <div
+            className={joinClassNames([
+                ' tw-flex tw-justify-center tw-items-center tw-absolute',
+                alignment === Alignment.Horizontal
+                    ? ' tw-h-full tw-w-[12px] -tw-translate-x-1/2 tw-cursor-ew-resize'
+                    : ' tw-w-full tw-h-[12px] -tw-translate-y-1/2 tw-cursor-ns-resize',
+            ])}
+        >
+            <div
+                style={{
+                    ...(getContrastStyle() as CSSProperties),
+                }}
```

Comment:
do you need to spread it? Couldn't it just be style={getContrastStyle}?

Reply:
Good catch, fixed